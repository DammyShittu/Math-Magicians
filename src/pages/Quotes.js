import React from 'react';

const Quotes = () => (
  <blockquote style={{ marginTop: '10%' }}>
    The definition of a good mathematical problem is the mathematics it generates, rather
    than the problem itself. ~Andrew Wiles
  </blockquote>
);

export default Quotes;
