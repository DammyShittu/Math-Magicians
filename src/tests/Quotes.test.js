import React from 'react';
import renderer from 'react-test-renderer';
import Quotes from '../pages/Quotes';

it('renders correctly', () => {
  const tree = renderer
    .create(<Quotes />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
