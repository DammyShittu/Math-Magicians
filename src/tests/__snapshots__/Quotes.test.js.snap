// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<blockquote
  style={
    Object {
      "marginTop": "10%",
    }
  }
>
  The definition of a good mathematical problem is the mathematics it generates, rather than the problem itself. ~Andrew Wiles
</blockquote>
`;
